src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { ApiError } from '../../types/ApiError';
+
+interface Props {
+  removeErrorMessage: () => void;
+  errorType: ApiError;
+}
+
+export const ErrorMessage: React.FC<Props> = ({
+  removeErrorMessage,
+  errorType,
+}) => {
+  let message = '';
+
+  switch (errorType) {
+    case ApiError.Get:
+      message = 'Unable to load todos';
+      break;
+    case ApiError.Post:
+      message = 'Unable to add todo';
+      break;
+    case ApiError.Patch:
+      message = 'Unable to update a todo';
+      break;
+    case ApiError.Delete:
+      message = 'Unable to delete a todo';
+      break;
+    default:
+      message = 'Something goes wrong';
+  }
+
+  return (
+    <div
+      className="
+        notification
+        is-danger
+        is-light
+        has-text-weight-normal"
+    >
```

Comment:
would be better to use `classNames` and add each style from new line as argument